clang/tools/clang-offload-wrapper/ClangOffloadWrapper.cpp

```diff
@@ -341,6 +377,39 @@ class BinaryWrapper {
   std::vector<std::string> TempFiles;
 
 private:
+  std::string SymPropBCFiles;
+  std::unique_ptr<util::SimpleTable> SymProp_Table{nullptr};
+  LLVMContext SymProps_C;
+  ExitOnError SymProps_ExitOnErr;
+  std::unique_ptr<Module> Current_SymProps_M{nullptr};
+  Constant *Current_DeviceImage_Initializer;
+
+  void Set_Current_DeviceImage_Initializer() {
```

Comment:
function name starts with lower case and should not have "_".

Reply:
Updated function name.